src/plone.server/plone/server/content.py

```diff
@@ -230,14 +242,14 @@ def __init__(self, context):
         self.context = context
 
     def supports(self, behavior_interface):
-        for behavior in self.enumerateBehaviors():
-            if behavior_interface in behavior.interface._implied:
-                return True
-        return False
+        """We support all behaviors that accomplish the for_."""
```

Comment:
We need to provide as plone.behavior asks it in the moment of adapting. Its great also that we have here in case we want to create a subadapter for specific content which policy may be different.